extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,355 @@
+
+* `@context` : a dictionary mapping XMP [namespaces](https://github.com/adobe/xmp-docs/blob/master/Namespaces.md) to the URI where they are defined.
+* `packets`: an array of metadata packets referencing the namespaces defined in the `namespaces` dictionary.
+
+The following example defines a glTF scene with a sample XMP metadata.
+
+```json
+"extensions": {
+    "KHR_xmp" : {
+        "@context" : {
+            "dc" : "http://purl.org/dc/elements/1.1/"
+        },
+        "packets" : [
+            {
+                "dc:contributor" : [ "Creator1Name", "Creator2Email@email.com", "Creator3Name<Email@email.com>"],
+                "dc:coverage" : "Bay Area, California, United States",
+                "dc:creator" : [ "CreatorName", "CreatorEmail@email.com"],
+                "dc:date" : ["1997-07-16T19:20:30+01:00"],
+                "dc:description" : { "en-us": "text"},
+                "dc:format" : "model/gltf-binary",
+                "dc:identifier" : "urn:stock-id:292930",
+                "dc:language" : ["en"],
+                "dc:publisher" : ["Company"],
+                "dc:relation" : ["https://www.khronos.org/"],
+                "dc:rights" : {"en-us": "BSD"},
+                "dc:source" : "http://related_resource.org/derived_from_this.gltf",
+                "dc:subject" : ["architecture"],
+                "dc:title" : { "en-us" : "MyModel", "it-it" : "Mio Modello"},
+                "dc:type" : ["Physical Object"],
+            }
+        ]
+    }
+}
+```
+
+## Instantiating XMP metadata
+
+Metadata can be instantiated in a gltf object by defining the `extensions.KHR_xmp` property and, within that, an index into the `packets` array using the `packet` property.
+The following example shows a glTF Mesh instantiating the XMP metadata at index `0`.
+
+```json
+{
+    "meshes": [
+      {
+        "extensions" : {
+            "KHR_xmp": {
+                "packet" : 0,
+            }
+        },
+        "primitives": [
+          {
+            "attributes": {
+                "NORMAL": 23,
+                "POSITION": 22,
+                "TANGENT": 24,
+                "TEXCOORD_0": 25
+            },
+            "indices": 21,
+            "material": 3,
+            "mode": 4
+          }
+        ]
+      }
+    ]
+}
+```
+
+### Notes:
+
+#### Precedence
+
+Metadata applied to JSON objects in the top level arrays ( `scenes`, `nodes`, `meshes`, `materials`, `images`, `animations`) has precedence over the metadata specified in the `asset` property of a glTF.
+A glTF might reference resources already containing XMP metadata. A relevant example would be an image object referencing a PNG/JPG file with embedded XMP metadata. The metadata embedded in the referenced resource takes precedence over the metadata specified in the glTF manifest. Note that glTF clients and viewers may ignore the metadata embedded in the referenced resources (for instance PNG/JPG images).
```

Comment:
Aren't the last two statements contradicting each other?

Reply:
I don't think it is a direct contradiction but I agree this might be a source of confusion. 
This was the result of the conversation we both had with @donmccurdy . 
I believe the conclusion was that: 
* we want nested objects and referenced resources to have the right to override the metadata defined lower in the hierarchy.
* to free clients from having to parse XML-XMP in nested resources. 

The resolution was a note directed to glTF editors to make sure that all relevant metadata in the glTF is defined in the glTF manifest (so that client can ignore metadata in embedded resources).
Does this sound like a good compromise to you? 
This is the point of the last sentence, should it be more explicit? I can add `Therefore we recommend glTF editors to make sure all relevant metadata in the glTF is defined in the glTF manifest`

Reply:
I got different conclusion from that conversation.
- Nested objects can override whatever they want as long as they keep everything in JSON.
- PNG/JPEG metadata is always ignored. Exporters should add metadata packets to `image` glTF objects if needed.

Reply:
Let me remove `The metadata embedded in the referenced resource takes precedence over the metadata specified in the glTF manifest.` I think it is indeed misleading.
`PNG/JPEG metadata is always ignored. Exporters should add metadata packets to image glTF objects if needed.` => yes this is what I was trying to say, I will make it more explicit.

Reply:
Hi @lexaknyazev, I believe I addressed your concerns, I am labelling this conversation resolved please double check and feel free to reopen/bring up additional concerns. Thank you.